### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by DamianAdamczewski**
- [DEMO LINK](https://DamianAdamczewski.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://DamianAdamczewski.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -9,6 +12,38 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <nav class="nav">
+        <img class="icon"
```
Comment (mykhalenych) [inline src/index.html:17]: Screenshot 2023-08-21 at 15 06 07

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,71 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
 }
+
+.nav {
+  /* border: 5px solid pink; */
```
Comment (mykhalenych) [inline src/style.css:7]: delete comments

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,71 @@
...
+
+a:active {
+  border-bottom-color: rgba(0, 172, 220, 1);
+}
+
+.icon {
+  height: 40px;
+}
+
+.rectangle {
```
Comment (mykhalenych) [inline src/style.css:56]: you don't have class rectangle

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,71 @@
...
+  list-style: none;
+  /* border: 5px solid blue; */
+  display: flex;
+  flex-grow: 2;
+  justify-content: flex-end;
+
+  margin: auto;
+}
+
+a {
```
Comment (mykhalenych) [inline src/style.css:29]: add classname for a tags and use .classname instead

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,71 @@
...
+  font-weight: 500;
+  /* border: 5px solid yellow; */
+  border-bottom: 4px solid transparent;
+}
+
+a:hover {
+  color: rgba(0, 172, 220, 1);
+}
+
+a:active {
```
Comment (mykhalenych) [inline src/style.css:48]: use .is-active::after and create line with position absolute, also add position relative for .is-active

**Review by mykhalenych — CHANGES_REQUESTED** [review]
Check comments, also check design
https://www.figma.com/file/1sog2rmfyCjnVxkeZ3ptnc/MOYO-%2F-Header?type=design&node-id=0-1&mode=design&t=Q95Hh3VXA4NIYJo9-0

File: src/index.html [hunk 6]
```diff
@@ -9,6 +12,78 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <main class="main">
+      <a href="#home">
+        <img
+      src="./images/logo.png"
+      alt="logo moyo"
+      >
```
Comment (OleziO) [inline src/index.html:20]:
```suggestion
        <img
          src="./images/logo.png"
          alt="logo moyo"
        >
```

File: src/index.html [hunk 7]
```diff
@@ -9,6 +12,78 @@
...
+    <header class="header">
+      <nav class="nav">
+        <ul class="nav__list">
+
+          <li class="nav__item">
+            <a href="#apple"
+             class="nav__link
+             is-active">
+             Apple
+            </a>
```
Comment (OleziO) [inline src/index.html:32]:
```suggestion
            <a 
              href="#apple"
              class="nav__link
              is-active"
            >
             Apple
           </a>
```

File: src/index.html [hunk 8]
```diff
@@ -9,6 +12,78 @@
...
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung"
+             class="nav__link">
```
Comment (OleziO) [inline src/index.html:37]:
```suggestion
            <a 
               href="#samsung"
               class="nav__link"
             >
```

Comment (OleziO) [reply src/index.html:37]: Check all other similar cases.

File: src/style.css [hunk 9]
```diff
@@ -1,3 +1,72 @@
...
+
+.nav__link:active::after {
+  content: "";
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 10px;
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc;
```
Comment (OleziO) [inline src/style.css:54]: Move all repeatable colors to the CSS variables.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)

**Review by OleziO — CHANGES_REQUESTED** [review]
Header must be full screen width.
image

**Review by plodnikate — CHANGES_REQUESTED** [review]
Good job, but use scss nesting and also check and follow approach ([CSS_order-rule](https://9elements.com/css-rule-order/))

File: src/index.html [hunk 10]
```diff
@@ -2,13 +2,85 @@
...
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
```
Comment (mykhalenych) [inline src/index.html:58]:
```suggestion
            <a
              href="#"
              class="nav__link"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
```

File: src/style.css [hunk 11]
```diff
@@ -1,3 +1,83 @@
...
+  line-height: 60px;
+  text-decoration: none;
+  color: var(--black-color);
+}
+
+.nav__link:hover {
+  color: var(--active-color);
+}
+
+.nav__link:active::after {
```
Comment (mykhalenych) [inline src/style.css:57]: it is redundant

**Review by mykhalenych — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 10 above)
Comment (DamianAdamczewski) [reply src/index.html:58]: done :)

**Review by mykhalenych — APPROVED** [review]
Nice work!

**Pull request closed without merging**